storage/videx/mysql-test/videx/r/create-table-and-index.result

```diff
@@ -0,0 +1,46 @@
+CREATE TABLE `orders` (
+`O_ORDERKEY` int NOT NULL,
+`O_CUSTKEY` int NOT NULL,
+`O_ORDERSTATUS` char(1) NOT NULL,
+`O_TOTALPRICE` decimal(15,2) NOT NULL,
+`O_ORDERDATE` date NOT NULL,
+`O_ORDERPRIORITY` char(15) NOT NULL,
+`O_CLERK` char(15) NOT NULL,
+`O_SHIPPRIORITY` int NOT NULL,
+`O_COMMENT` varchar(79) NOT NULL,
+PRIMARY KEY (`O_ORDERKEY`),
+KEY `ORDERS_FK1` (`O_CUSTKEY`)
+) ENGINE=VIDEX;
+SHOW CREATE TABLE orders;
+Table	Create Table
+orders	CREATE TABLE `orders` (
+  `O_ORDERKEY` int(11) NOT NULL,
+  `O_CUSTKEY` int(11) NOT NULL,
+  `O_ORDERSTATUS` char(1) NOT NULL,
+  `O_TOTALPRICE` decimal(15,2) NOT NULL,
+  `O_ORDERDATE` date NOT NULL,
+  `O_ORDERPRIORITY` char(15) NOT NULL,
+  `O_CLERK` char(15) NOT NULL,
+  `O_SHIPPRIORITY` int(11) NOT NULL,
+  `O_COMMENT` varchar(79) NOT NULL,
+  PRIMARY KEY (`O_ORDERKEY`),
+  KEY `ORDERS_FK1` (`O_CUSTKEY`)
+) ENGINE=VIDEX DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_uca1400_ai_ci
+CREATE INDEX idx_orders_status_date ON orders (O_ORDERSTATUS, O_ORDERDATE);
+SHOW CREATE TABLE orders;
+Table	Create Table
+orders	CREATE TABLE `orders` (
+  `O_ORDERKEY` int(11) NOT NULL,
+  `O_CUSTKEY` int(11) NOT NULL,
+  `O_ORDERSTATUS` char(1) NOT NULL,
+  `O_TOTALPRICE` decimal(15,2) NOT NULL,
+  `O_ORDERDATE` date NOT NULL,
+  `O_ORDERPRIORITY` char(15) NOT NULL,
+  `O_CLERK` char(15) NOT NULL,
+  `O_SHIPPRIORITY` int(11) NOT NULL,
+  `O_COMMENT` varchar(79) NOT NULL,
+  PRIMARY KEY (`O_ORDERKEY`),
+  KEY `ORDERS_FK1` (`O_CUSTKEY`),
+  KEY `idx_orders_status_date` (`O_ORDERSTATUS`,`O_ORDERDATE`)
+) ENGINE=VIDEX DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_uca1400_ai_ci
+DROP TABLE orders;
```

Comment:
Up to you, but you don't have to have separate `r/` and `t/` directories. You can have these files under `videx` if you like, e.g.:
```
storage/videx/mysql-test/videx/create-table-and-index.result
storage/videx/mysql-test/videx/create-table-and-index.test
```